eclipse-che/che-operator
# feat: Configure external devfile registries urls

Signed-off-by: Anatolii Bazko <abazko@redhat.com>



### What does this PR do?
- `status.DevfileRegistryURL` contain only url to registry deployed by operator (not external ones)
- `CHE_WORKSPACE_DEVFILE__REGISTRY__URL` contains all urls including external ones

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/19781

### How to test this PR?

1. Deploy Eclipse Che
2. Configure `spec.server.externalDevfilerRegistries`
3. Check `CHE_WORKSPACE_DEVFILE__REGISTRY__URL` inside `che` config map. It must contain internal + external devfile registries urls

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
